framework/src/utils/PerfGraph.C

```diff
@@ -21,10 +21,13 @@
 // System Includes
 #include <chrono>
 
-PerfGraph::PerfGraph() : _current_position(0), _active(true)
+const std::string PerfGraph::ROOT_NAME = "Root";
```

Comment:
I don't see a purpose in defining this - you didn't even use it in the error messages below (although I don't think it's necessary to use it in the error messages).